build/target-repository/bootstrap.php

```diff
@@ -2,69 +2,80 @@
+                $composerAutoloader = require __DIR__ . '/vendor/autoload.php';
+            }
+            $composerAutoloader->loadClass($class);
         }
-        $composerAutoloader->loadClass($class);
-    }
 
-    // aliased by php-scoper, that's why its missing
-    if ($class === 'Symplify\SmartFileSystem\SmartFileInfo') {
-        $filePath = __DIR__ . '/vendor/symplify/smart-file-system/src/SmartFileInfo.php';
-        if (file_exists($filePath)) {
-            require $filePath;
+        // aliased by php-scoper, that's why its missing
+        if ($class === 'Symplify\SmartFileSystem\SmartFileInfo') {
+            $filePath = __DIR__ . '/vendor/symplify/smart-file-system/src/SmartFileInfo.php';
+            if (file_exists($filePath)) {
+                require $filePath;
+            }
         }
-    }
 
-    if ($class === 'Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator') {
-        // avoid duplicated autoload bug in Rector demo runner
-        if (class_exists('Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator', false)) {
-            return;
+        if ($class === 'Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator') {
+            // avoid duplicated autoload bug in Rector demo runner
+            if (class_exists('Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator', false)) {
+                return;
+            }
         }
-    }
-});
+    });
 
-if (! interface_exists('UnitEnum')) {
-    /**
-     * @since 8.1
-     */
-    interface UnitEnum
-    {
+    if (! interface_exists('UnitEnum')) {
         /**
-         * @return static[]
+         * @since 8.1
          */
-        public static function cases(): array;
+        interface UnitEnum
+        {
+            /**
+             * @return static[]
+             */
+            public static function cases(): array;
+        }
     }
-}
 
-if (! interface_exists('BackedEnum')) {
-    /**
-     * @since 8.1
-     */
-    interface BackedEnum extends UnitEnum {
+    if (! interface_exists('BackedEnum')) {
         /**
-         * @param int|string $value
-         * @return $this
+         * @since 8.1
          */
-        public static function from($value);
+        interface BackedEnum extends UnitEnum {
+            /**
+             * @param int|string $value
+             * @return $this
+             */
+            public static function from($value);
 
-        /**
-         * @param int|string $value
-         * @return $this|null
-         */
-        public static function tryFrom($value);
+            /**
+             * @param int|string $value
+             * @return $this|null
+             */
+            public static function tryFrom($value);
+        }
+    }
+}
+
+namespace PHPUnit\Framework
+{
+    if (! class_exists('PHPUnit\Framework\TestCase')) {
+        abstract class TestCase
+        {
+        }
     }
 }
```

Comment:
Could you link the original issue, so we know why we use this hack?

Reply:
added https://github.com/rectorphp/rector-src/pull/1924/commits/553d54b6a5b4fb8416b5e06a048f3a9d2a1be06b for `@see` to original issue.